oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -105,8 +105,9 @@ int pldm::responder::oem_ibm_platform::Handler::
 
         else
         {
-            error("Invalid entity type received. Entity type = {ENTITY_TYP}",
-                  "ENTITY_TYP", entityType);
+            error(
+                "Invalid entity type received by function oemSetNumericEffecterValueHandler. Entity type = {ENTITY_TYP}",
+                "ENTITY_TYP", entityType);
```

Comment:
can we start logging error traces as per andrews comment from here onwards?
please go through all comment and apply here.
[](https://gerrit.openbmc.org/c/openbmc/pldm/+/67994)

Reply:
Done